e2e-tests/cypress/integration/appellant-confirms-declaration.feature

```diff
@@ -0,0 +1,12 @@
+@UI-ONLY
+Feature: Declaration must be agreed before a submission can be made
+
+  Scenario: Prospective appellant does not agree to the declaration
+    Given an appeal wants to be submitted
```

Comment:
phrasing?
it could be ok, but that feels like quite a jump- starting to give wants+needs to the appeal itself.
given an appeal has been prepared? given a prospective appellant has completed their appeal details?

(not a problem in terms of it's ability to prove what's going on, just not sure that's the phrasing we'd have chosen?)